src/theme/theme.js

```diff
@@ -5,21 +5,22 @@ const theme = {
     mediumGrey: '#E1E1E1',
```

Comment:
![image](https://user-images.githubusercontent.com/17087167/41498051-698cc8da-715b-11e8-9499-9d820efeacc5.png)

updated the colours variables so my IDE looks like this